docs/secure-connections/sauce-connect/advanced/sc-kgp.md

```diff
@@ -0,0 +1,46 @@
+---
+id: sc-kgp
+title: KGP
+sidebar_label: Tunneling Protocol
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Overview
+
+This document provides information about KGP - the tunneling protocol used by Sauce Connect Proxy&trade;
```

Comment:
What does KGP stand for?

Reply:
oh god :). It stands for "Keep-Going Protocol", it was a joke.